src/docs/product/integrations/integration-platform/internal-integration.mdx

```diff
@@ -4,6 +4,12 @@ sidebar_order: 2
 description: "Learn more about Sentry's internal integrations."
 ---
 
+<Note>
+
+We've built a [reference implementation](https://github.com/getsentry/integration-platform-example) to help you get started with building on Sentry's Integration Platform! Check it out for realistic code examples for these adding features in Python and TypeScript. See the [quick start](/product/integrations/integration-platform/#quick-start) for more info.
+
+</Note>
```

Comment:
I would have liked to reuse this across the pages, but I don't think you can nest the Markdown Renderer in itself :( At least I couldnt figure out a way to do it.

Reply:
E.g. If I make a `<ReferenceImplAlert />` component, I couldn't use markdown syntax within it, and would have to set up HTML tags like <a> myself, but they might not get the rich styling/attributes that the MDX rendered HTML tags receive